### PR TITLE
Create CleanerRobot.md

### PR DESCRIPTION
**Pull request by ftrotter**
I added a new cleaner robot! It should be added to main!!

### CODE REVIEW

File: CleanerRobot.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Robot Name: Cleaner Robot
```
Comment (Kianadesme) [inline CleanerRobot.md]: This needs a hashtag for markdown

File: CleanerRobot.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Function: Sweep Floor Really Well
```
Comment (Kianadesme) [inline CleanerRobot.md]: This and all of the other functions should be renamed as "tasks".

**Review by Kianadesme — CHANGES_REQUESTED** [review]
Not good enough!

File: CleanerRobot.md (hunk 1 above)
Comment (ftrotter) [reply CleanerRobot.md]: changed

File: CleanerRobot.md (hunk 2 above)
Comment (ftrotter) [reply CleanerRobot.md]: changed

**Comment by ftrotter** [comment]
I think I have added what I needed, could you take another look?

**Review by Kianadesme — COMMENTED** [review]
I saw that you added a new task to wipe the seats, but maybe we should just say "Wipe down store" and have one task. What do you think?

**Comment by ftrotter** [comment]
I think for now, we should have two seperate tasks. Lets start there and see how it goes.

**Review by Kianadesme — APPROVED** [review]
Okay, it looks good.

**Pull request merged**